packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/configurations/ModelConfiguration.java

```diff
@@ -23,6 +23,9 @@ public class ModelConfiguration extends TerariumAsset {
 	private UUID calibrationRunId;
 	private UUID modelId;
 
+	/** This is "simulation" in the sense of our POJO. It actually corresponds to a pyciemss calibration */
+	private UUID simulationId;
```

Comment:
Probably optional and/or tsoptional? Doe does it already default to null in hibernate land?

Reply:
oh good catch - yes it does